packages/hopli/src/network_registry.rs

```diff
@@ -0,0 +1,71 @@
+use crate::process::{child_process_call_foundry_self_register, set_process_path_env};
+use clap::Parser;
+
+use crate::utils::{Cmd, HelperErrors};
+
+/// CLI arguments for `hopli network-registry`
+#[derive(Parser, Default, Debug)]
+pub struct NetworkRegistryArgs {
```

Comment:
The command should be more description, like `register-in-network-registry`.

Reply:
Updated